src/components/Drawer.js

```diff
@@ -131,31 +126,31 @@ const Drawer = React.createClass({
           type='base'
         />
       </nav>
-    ) : <div style={styles.nav} />;
+    ) : <div className={css(styles.nav)} />;
   },
 
   render () {
     const styles = this.styles();
 
     return (
-      <div style={styles.componentWrapper}>
-        <div onClick={this.close} style={styles.scrim} />
-        <div ref={(ref) => (this._component = ref)} style={Object.assign({}, styles.component, this.props.style)}>
-          <header style={Object.assign({}, styles.header, this.props.headerStyle)}>
-            <span style={styles.backArrow}>
+      <div className={css(styles.componentWrapper)}>
+        <div className={css(styles.scrim)} onClick={this.close} />
+        <div className={css(styles.component)} ref={(ref) => (this._component = ref)}>
+          <header className={css(styles.header)}>
+            <span className={css(styles.backArrow)}>
               <Button
                 icon='arrow-left'
                 onClick={this.close}
                 primaryColor={this.props.buttonPrimaryColor}
                 type={'base'}
               />
             </span>
-            <span style={styles.title}>
+            <span className={css(styles.title)}>
               {this.props.title}
             </span>
             {this._renderNav()}
           </header>
-          <div style={Object.assign({}, styles.content, this.props.contentStyle)}>
+          <div className={css(styles.content)}>
```

Comment:
👍  I like that you moved the concern of merging this and the other style overrides into `styles`, I think it makes the jsx cleaner and when looking at the styles it's apparent which can be overridden.